CodeModel/CodeParsers/CSharp/ProjectAnalyzer/Access.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace CodeModel.CodeParsers.CSharp.ProjectAnalyzer
```

Comment:
Currently, the cohesion analyzer is inside the `ProjectAnalyzer` directory. I was thinking of renaming the directory to `ClassCohesionAnalyzer`. What do you think?